tests/reporter/kubernetes.go

```diff
@@ -361,3 +476,20 @@ func (r *KubernetesReporter) AfterSuiteDidRun(setupSummary *types.SetupSummary)
 func (r *KubernetesReporter) SpecSuiteDidEnd(summary *types.SuiteSummary) {
 
 }
+
+//getInterestingNodes returns all node where a virt-launcher pod ran (finished) or still runs
+func getInterestingNodes(virtCli kubecli.KubevirtClient) map[string]struct{} {
```

Comment:
can this be `getVirtualLauncherNodes` or something along that line?
Also, can it return a slice instead of a map?

Reply:
Done.